Directory.Build.props

```diff
@@ -27,8 +26,8 @@
 
```

Comment:
In line 22 `RootIntermediateOutputPath` is still defined. I thought with the buildtools removal we don't need it anymore?

Reply:
Good catch, I'll delete it